### PR TITLE
Deeplink and import from library changes

### PR DESCRIPTION
**Pull request by Vinod-V3**

### CODE REVIEW

File: src/app/project-library/selected-categories-list/selected-categories-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  constructor(private navCtrl: NavController, private toastService: ToastService, private activatedRoute: ActivatedRoute, private router: Router) {
+    this.apiService = inject(ApiBaseService)
+    activatedRoute.params.subscribe((param:any)=>{
+      this.selectedCategory = param['category']
+    })
+    this.stateData = this.router.getCurrentNavigation()?.extras.state
+  }
+
+  ngOnInit() {
```
Comment (5Amogh) [inline src/app/project-library/selected-categories-list/selected-categories-list.component.ts:30]: we can remove this ununsed method

File: src/app/project/project-details.page.ts [hunk 2]
```diff
@@ -40,4 +33,15 @@ export class ProjectDetailsPage  implements OnInit {
     goBack(){
       this.navCtrl.back();
     }
+
+    getProfileDetails() {
+      this.profileService.getProfileAndEntityConfigData().subscribe((mappedIds) => {
+        if (mappedIds) {
+          this.config.profileInfo = mappedIds;
+          this.showDetails = true
```
Comment (5Amogh) [inline src/app/project/project-details.page.ts:41]: in both case the showdetails is true so we can put this outside if-else blocks

File: src/app/services/profile/profile.service.ts [hunk 3]
```diff
@@ -74,15 +74,16 @@ export class ProfileService {
...
-      if (key === 'roles' && data.user_roles) {
-        result[key] = data.user_roles.map((role: any) => role.title).join(',');
-      } else if (data[key]) {
-        if (Array.isArray(data[key])) {
-          result[key] = data[key].map((item: any) => item).join(',');
-        } else if (data[key].value) {
-          result[key] = data[key].value;
-        }
-      }
+      // if (key === 'roles' && data.user_roles) {
```
Comment (5Amogh) [inline src/app/services/profile/profile.service.ts:77]: we can  remove the commented code

File: src/app/services/util/util.service.ts [hunk 4]
```diff
@@ -12,6 +12,12 @@ export class UtilService {
   isMobile(){
     return /iPhone|iPad|iPod|Android/i.test(window.navigator.userAgent);
   }
+
+  isLoggedIn(){
+    let token = localStorage.getItem('accToken')
```
Comment (5Amogh) [inline src/app/services/util/util.service.ts:17]: we can use tertiary operator to reduce this to single liner code

**Review by 5Amogh — CHANGES_REQUESTED** [review]

File: src/app/services/profile/profile.service.ts (hunk 3 above)
Comment (Vinod-V3) [reply src/app/services/profile/profile.service.ts:77]: The commented code is required for next release

**Pull request merged**